<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabAppearance">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_MapPic">
         <property name="title">
          <string>Map</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_MapPic"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_MapPic">
            <property name="text">
             <string>Choose Picture</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_RobotPic">
         <property name="title">
          <string>Robot</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="comboBox_RP_Shape"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_RP_Size"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>px</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_RP_Color">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>320</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSerial">
      <attribute name="title">
       <string>Serial</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="selectBox">
         <property name="title">
          <string>Select Serial Port</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QComboBox" name="serialPortInfoListBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="manufacturerLabel">
            <property name="text">
             <string>Manufacturer:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="serialNumberLabel">
            <property name="text">
             <string>Serial number:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="locationLabel">
            <property name="text">
             <string>Location:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="vidLabel">
            <property name="text">
             <string>Vendor ID:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="pidLabel">
            <property name="text">
             <string>Product ID:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="parametersBox">
         <property name="title">
          <string>Select Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="baudRateLabel">
            <property name="text">
             <string>BaudRate:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="baudRateBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dataBitsLabel">
            <property name="text">
             <string>Data bits:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="dataBitsBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="parityLabel">
            <property name="text">
             <string>Parity:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="parityBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="stopBitsLabel">
            <property name="text">
             <string>Stop bits:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="stopBitsBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="flowControlLabel">
            <property name="text">
             <string>Flow control:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="flowControlBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>96</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
